com.unity.multiplayer.mlapi/Runtime/Messaging/MessageBatcher.cs

```diff
@@ -126,40 +137,55 @@ public void QueueItem(in RpcFrameQueueItem queueItem)
                 if (sendStream.IsEmpty)
                 {
                     sendStream.IsEmpty = false;
-                    sendStream.NetworkChannel = queueItem.NetworkChannel;
-
-                    switch (queueItem.QueueItemType)
-                    {
-                        // 8 bits are used for the message type, which is an NetworkConstants
-                        case RpcQueueContainer.QueueItemType.ServerRpc:
-                            sendStream.Writer.WriteByte(NetworkConstants.SERVER_RPC); // MessageType
```

Comment:
Is it that we no longer need to send the server / client RPC stuff because the `EnterInternalCommandContext` takes this for us?  If so, cool, cleaner.

Reply:
The queue stuff was using two different message types, the QueueItemType and the NetworkConstants type. This code used to translate from the former to the latter, but when I was moving everything over it felt really redundant to have two different sets of constants containing the exact same list of items, so I got rid of the NetworkConstants and put everything in the MessageType enum. Since the MessageType enum is what we're sending now, there's no longer a need to translate it to a different set of constants.

Reply:
Nice